app/scripts/views/components/barChart/barChart.js

```diff
@@ -363,6 +413,18 @@
         } else if (max < 100 &&
           max > 10) {
           config.divider = 2;
+        } else if (data.smallDataFlag) {
+          if (exponentRange > 20) {
+            config.divider = 8;
```

Comment:
if (data.smallDataFlag) {
          var numberOfTickValues = 4;
          config.divider = Math.round(exponentRange / numberOfTickValues);
}
Now config.divider is calculated automatically.